src/pipecat/transports/tavus/transport.py

```diff
@@ -274,6 +275,10 @@ async def _on_left(self):
         """Handle left event."""
         logger.debug("TavusTransportClient left!")
 
+    async def _on_before_leave(self):
+        """Handle before leave event."""
+        logger.debug("TavusTransportClient before leave!")
```

Comment:
This just follows the pattern in this file, of logging when we don't actually need to do anything for a `DailyCallbacks` callback